projects/storefrontstyles/scss/themes/calydon/layout/page-templates/_category-page.scss

```diff
@@ -0,0 +1,11 @@
+%CategoryPageTemplate-calydon {
+  cx-page-slot {
+    cx-banner {
+      a:focus {
+        display: block;
+        position: sticky;
```

Comment:
we shouldn't use something that experimental:
https://caniuse.com/#feat=css-sticky

+ same comment on playing with display prop on focus

Reply:
i know... one of those crazy fixes, as you may have noticed while browsing, it seems like the only solution that makes the z-index kick-in without visual damages to punch out the element and display the focus outline. Suggestions welcome.

Reply:
never mind it also works with `position:relative`